scene/2d/tile_map.cpp

```diff
@@ -2024,6 +2024,17 @@ void TileMap::set_cell(int p_layer, const Vector2i &p_coords, int p_source_id, c
 		ERR_FAIL_COND(!Q);
 		TileMapQuadrant &q = Q->value;
 
+		// Find node in scenes and remove it.
+		auto entry = q.scenes.find(pk);
```

Comment:
Usage of `auto` is [not allowed](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#auto-keyword)